config/api.yml

```diff
@@ -2487,6 +2487,8 @@
         :identifier: catalogitem_delete
       - :name: create
         :identifier: atomic_catalogitem_new
+      - :name: order
+        :identifier: svc_catalog_provision
```

Comment:
don't we need the :validate_action option here too ?

Reply:
@abellotti it doesn't go on the collection, only on the resource, which is consistent with the other places we have it

Reply:
👍